frontend/src/components/search/SearchResultsList.tsx

```diff
@@ -37,6 +37,10 @@ const SearchResultsList: React.FC<SearchResultsListProps> = ({
               <div className="grid-row flex-column">
                 <div className="grid-col tablet:order-2">
                   <h2 className="margin-y-105 line-height-serif-2">
+                    {/* TODO: href here needs to be set to:
+                        dev/staging: https://grants.gov/search-results-detail/<opportunity_id>
+                        local/prod: https://grants.gov/search-results-detail/<opportunity_id>
+                    */}
```

Comment:
Just a note to set the link to the actual grant/opportunity - it differs for each environment

Reply:
FWIW, this is the same URL twice. eg. `https://grants.gov/search-results-detail/<opportunity_id>`